pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -342,8 +356,8 @@ func (p *AnacondaInstallerISOTree) serialize() osbuild.Pipeline {
 	default:
 	}
 
-	if p.ISOLinux {
-		isoLinuxOptions := &osbuild.ISOLinuxStageOptions{
+	if p.ISOBoot == SyslinuxISOBoot {
```

Comment:
And then I assume we don't need to do anything extra for the grub2 UEFI only version?